metadata-ingestion/tests/integration/powerbi/golden_test_admin_only.json

```diff
@@ -1214,18 +1214,44 @@
         "json": [
             {
                 "op": "add",
-                "path": "/customProperties/chartCount",
-                "value": "2"
+                "path": "/charts",
+                "value": [
+                    "urn:li:chart:(powerbi,charts.B8E293DC-0C83-4AA0-9BB9-0A8738DF24A0)",
+                    "urn:li:chart:(powerbi,charts.23212598-23b5-4980-87cc-5fc0ecd84385)"
+                ]
             },
             {
                 "op": "add",
-                "path": "/customProperties/workspaceName",
-                "value": "demo-workspace"
+                "path": "/customProperties",
+                "value": {
+                    "chartCount": "2",
+                    "workspaceName": "demo-workspace",
+                    "workspaceId": "64ED5CAD-7C10-4684-8180-826122881108"
```

Comment:
patches have very specific formats, and I don't think GMS will actually accept this formatting correctly

Reply:
updated the implementation